missing/strlcpy.c

```diff
@@ -1,4 +1,4 @@
-/*	$OpenBSD: pcap_strlcpy.c,v 1.12 2015/01/15 03:54:12 millert Exp $	*/
+/*	$OpenBSD: strlcpy.c,v 1.12 2015/01/15 03:54:12 millert Exp $	*/
```

Comment:
Search and replace misfired here (possibly somewhere else too).

Reply:
Nope, that's a fix to an earlier search-and-replace from a while ago that shouldn't have happened in the first place; we (somebody, probably me) grabbed the OpenBSD strlcpy.c, renamed the routine, and added it.  I fixed the /* OpenBSD */ comment to use the original source file's name.

Reply:
I stand corrected.

Reply:
It was probably my screwup in the first place, so don't worry about it.